src/common/sms/smsUtils.js

```diff
@@ -0,0 +1,51 @@
+export const formatSecretValue = async (secretValue) => {
+  if (!secretValue) {
+    return '';
+  }
+
+  if (globalThis.CryptoKey && secretValue instanceof CryptoKey) {
+    return await formatCryptoKey(secretValue);
+  }
```

Comment:
Though this will not work, we may need to check with node [`util.types.isCryptoKey(value)`](https://nodejs.org/docs/latest-v18.x/api/util.html#utiltypesiscryptokeyvalue), wich will need to be dynamically imported...